galleries/plot_types/3D/scatter3d_simple.py

```diff
@@ -1,6 +1,6 @@
 """
 ==============
-3D scatterplot
+scatter(x, y, z)
 ==============
```

Comment:
The `=` above and below need to be the same length as the heading text, in all cases.